src/components/util/AutocompleteInput/AutocompleteList.tsx

```diff
@@ -157,9 +157,7 @@ export const AutocompleteList = React.forwardRef<HTMLDivElement, AutocompleteLis
 		}, [handleKeyDown]);
 
 		React.useEffect(() => {
-			const showAutocomplete = Boolean(
-				(alwaysShow || (anchor && value && value !== focusedOption)) && list.length,
-			);
+			const showAutocomplete = Boolean((alwaysShow || (anchor && value)) && list.length);
```

Comment:
why was this change required?
There is issue now when there is only one item in autocomplete list, you try to click it, and autocomplete list is still showing. But i would expect it to close

Reply:
that s why
![image](https://user-images.githubusercontent.com/74649472/144998276-c6586446-5197-4de8-8eef-ac6bc48ff70b.png)